src/components/request/useSettledTableData.tsx

```diff
@@ -3,6 +3,11 @@ import { ICell, IRow } from "../table/Table";
 import { oracle } from "@uma/sdk";
 import { ethers } from "ethers";
 import { ChainId } from "constants/blockchain";
+import {
+  ActiveRequestParams,
+  ActiveRequestParamsByTransaction,
+} from "hooks/useRequestParams";
+import formatYesNoQueryString from "helpers/formatYesNoQueryString";
```

Comment:
i think theres already a formatting file in utils, can we just add it there rather than make its own file

Reply:
Sure, we can combine formatting functions into 1 bigger file.